### ISSUE TITLE
Moderate legislation proposals

### ISSUE THREAD

**Issue by decabeza** [consul/consul#2940]
## Background

Admins can moderate the content on show views _[see screenshot]_. But on legislation proposals `/legislation/processes/N/proposals/N` it is not working.

## What we need
Fix this feature to hide legislation proposals.

## Screenshots
![hide legislationsproposals](https://user-images.githubusercontent.com/631897/46287278-4f339100-c582-11e8-8243-e2eb0a96d15e.png)

## Notes

It seems related with:
```erb
<%= link_to t("admin.actions.hide").capitalize, hide_moderation_proposal_path(proposal),
              method: :put, remote: true, data: { confirm: t('admin.actions.confirm') } %>
```

on `views/legislation/proposals/_actions.html.erb`

This `hide_moderation_proposal_path(proposal)` is generating the incorrect url `/moderation/proposals/N-proposal-title/hide` instead of `/moderation/legislation/processes/N/proposals/N-proposal-title/hide`

**Comment by cyrillefr** [comment]
Hello, Ola

I have seen this issue a while ago and started working on it this week-end.
I think I have a solution, so I post it here, even though I did not ask to work on it. 

NB: After the legislation being moderated, nothing happens, but it appears normal since there is no redirection.
But, one can check in the database that the hidden_at field is not null anymore.(There is also a spec).

NB2: In the Administration interface, there is no menu for legislation proposals to be unmoderated(might need an issue/PR for that).

**Comment by javierm** [comment]
Closed via #3602.
